homeassistant/components/fullykiosk/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""The Fully Kiosk Browser integration."""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DEFAULT_PORT, DOMAIN
+from .coordinator import FullyKioskDataUpdateCoordinator
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+PLATFORMS = [
+    "binary_sensor",
+    "sensor",
```

Comment:
```suggestion
    Platform.BINARY_SENSOR,
    Platform.SENSOR,
```

Reply:
That said, please limit the PR to a single platform.
It really makes things easier in follow up reviews (and this review).